docs/fundamentals.txt

```diff
@@ -15,13 +15,15 @@ Fundamentals
    :titlesonly:
    :maxdepth: 1
 
+   /fundamentals/connection
    /fundamentals/database-collection
    /fundamentals/read-operations
    /fundamentals/write-operations
 
 Learn how to use the {+odm-long+} to perform the following tasks:
 
-- :ref:`Manage Databases and Collections <laravel-db-coll>`
+- :ref:`Connections <laravel-fundamentals-connection>`
```

Comment:
S: also add this to the list of fundamentals pages on the index (alternatively, move into an include that is used on both pages)

Reply:
Adding the item to index.txt and then adding the task to move it into an include to the cleanup ticket since it impacts other content.